src/DrugstoreApp/Backend/DrugstoreAPI/DrugstoreAPI/Controllers/DrugstoreFeedbackWithAPIController.cs

```diff
@@ -0,0 +1,41 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using DrugstoreAPI.Filters;
+using RestSharp;
+
+namespace DrugstoreAPI.Controllers
+{
+    [Route("api/[controller]")] // /api/DrugstoreFeedbackWithAPI
+    [ApiController]
+    [ApiKeyAuth]
+    public class DrugstoreFeedbackWithAPIController : ControllerBase
+    {
+        [HttpGet(template: "secret")]
+        public IActionResult TestAPI()
+        {
+            return Ok("You can use our drugstore services :)");
+        }
+
+        [HttpGet(template: "hitHospital")]
+        public IActionResult HitHospital()
+        {
+            var client = new RestSharp.RestClient("http://localhost:5000");
```

Comment:
Transfer into application.settings because of docker.

Reply:
Sredjeno!